src/index.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+// C = K - 273.15
+
+const state = {
+  tempCount: 70,
+  cityName: "",
+  tempValue: null,
+};
+
+//wave 2
+const updateDecreaseTempCount = () => {
+  --state.tempCount;
+  refreshUI()
+  updateTempColorAndLandscape();
+};
+
+const updateIncreaseTempCount = () => {
+  ++state.tempCount;
+  refreshUI();
+  updateTempColorAndLandscape();
+};
+
+const refreshUI = () => {
+  state.tempValue.textContent = `${state.tempCount}°F`;
+};
```

Comment:
Great work on these functions! The only thing I would change is the `refreshUI` function name. This name makes think that it would be refreshing the entire page when in actuality it is simply updating the text content for the element displaying the temperature.